sources/packages/web/src/components/common/StudentApplication.vue

```diff
@@ -19,6 +19,30 @@
     secondaryLabel="Previous section"
     class="mx-0"
   >
+    <template #primary-buttons v-if="!readonly && isLastPage">
```

Comment:
There's no `readonly`:
![image](https://github.com/user-attachments/assets/5596a2c7-c43d-4a6d-862a-2d8ff85c93a0)

Reply:
It seems to be a typo, the property is rather called, `isReadOnly`